src/intro.md

```diff
@@ -13,8 +13,8 @@ This is the official “LinuxBoot Book” for the LinuxBoot project. The book:
 LinuxBoot is the idea of replacing proprietary or corporate-driven late-stage
 boot [firmware](./glossary.md) with the Linux kernel and a community-based
 user-space. That idea grew into a project that over the years includes various
-initiatives with the overarching goal of moving from obscure and complex
-firmware to simpler and open source firmware.
+initiatives with the overarching goal of moving from obscure and closed source
+firmware to clear and open source firmware.
```

Comment:
It is currently a bit of a weird paragraph, I imagine from trying to rework the original text. Basically it reads like, "There is this idea, that spawned a project that focused on the idea", and thereby defines the idea twice slightly differently.

Maybe just keep, "That idea grew into a project that over the years includes various initiatives." And remove the rest of the line. And perhaps update the first sentence to include "open source".